modules/ROOT/pages/apikit-4-implement-rest-api.adoc

```diff
@@ -0,0 +1,257 @@
+        "lng": "81.1496"
+      }
+    },
+    "phone": "1-770-736-8031 x56442",
+    "website": "hildegard.org",
+    "company": {
+      "name": "Romaguera-Crona",
+      "catchPhrase": "Multi-layered client-server neural-net",
+      "bs": "harness real-time e-markets"
+    }
+  }
+]
+----
+. Save it as `user-example.json`.
+. Create a new blank file with the following API specification:
++
+[source,yaml,linenums]
+--
+#%RAML 1.0
+title: placeholder
+version: 1.0
+/users:
+  get:
+    description: Retrieve a list of all the users
+    responses:
+      200:
+        body:
+          application/json:
+            example: !include user-example.json
+  /userbyid:
+    get:
+      description: Get information about a particular user
+      queryParameters:
+        id:
+          description: Specify the id of the user you want to retrieve
+          type: integer
+          required: false
+          example: 3
+      responses:
+        200:
+          body:
+            application/json:
+              example: |
+                [
+                  {
+                    "id": 3,
+                    "name": "Clementine Bauch",
+                    "username": "Samantha",
+                    "email": "Nathan@yesenia.net",
+                    "address":
+                      {
+                        "street": "Douglas Extension",
+                        "suite": "Suite 847",
+                        "city": "McKenziehaven",
+                        "zipcode": "59590-4157",
+                        "geo": { "lat": "-68.6102", "lng": "-47.0653" },
+                      },
+                    "phone": "1-463-123-4447",
+                    "website": "ramiro.info",
+                    "company":
+                      {
+                        "name": "Romaguera-Jacobson",
+                        "catchPhrase": "Face to face bifurcated interface",
+                        "bs": "e-enable strategic applications",
+                      }
+                  }
+                ]
+--
+. Save it as `api.raml`.
+. In Studio, select *File* > *New* > *Mule Project*.
+. In *Project Name*, type the name of your Mule project.
+* If you are importing your API specification from Exchange, select *Import a published API*, click the plus icon, and select *From Exchange*.
++
+image::import-from-exchange.png["*Import a published API* highlighted when clicking on plus to select the From Exchange option"]
+
+* If you are importing your API specification from a local file in your file system, select *Import RAML from Local File*.
++
+image::import-from-local-file.png["*Import RAML from local file* highlighted in the API implementation section"]
+* If you are importing your API specification from Design Center, select *Download RAML from Design Center*.
++
+image::import-from-design-center.png["*Import from Design Center* highlighted in the API implementation section"]
+* Select *Scaffold flows from these API specifications* if not already selected.
```

Comment:
```suggestion
. Select *Scaffold flows from these API specifications* if not already selected.
```
I think this is a general step and not one of the options from the previous steps right? Please check.

Reply:
You're right! Great catch! Thanks!